akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1164,6 +1183,7 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef, joinConfigCompatCh
 
         if (leaderActionCounter == firstNotice || leaderActionCounter % periodicNotice == 0)
           logInfo(
+            ClusterLogMarker.leaderDetained,
```

Comment:
`detained` isn't that when someone is put in jail?

Reply:
ok, I looked for a word but probably wrong if it can be associated with that.
`leaderStalled`?

Reply:
I'll use my third lifeline, "call a friend". @chbatey ?

Reply:
changed to Incapacitated